### PR TITLE
Added confirmation dialog for submitting assessment

### PR DESCRIPTION
**Pull request by maxr96**
### Checklist
- [X] I run `yarn run webpack:build:main`: the project builds without errors.
- [X] I run `yarn lint`: the project builds without code style warnings.
- [X]  I tested the changes and all related features on the test server https://artemistest.ase.in.tum.de.

### Motivation and Context

This should resolve the issue #355.

### Description

I added a window submit, if user didn't assess all UML elements. If he then decides to close it, not assessed elements are highlighted. 

### Steps for Testing

1. Log in to ArTEMiS
2. Navigate to Course Administration
3. Select Tutor Course Dashboard
4. Select Exercise Dashboard for any modelling exercise
5. Select Open Assessment
6. Click Submit with no or part of the elements assessed to see confirmation dialog

### Screenshots

Partially asssessed
Warning dialog
Unassesed elements highlighted

### CODE REVIEW

File: src/main/webapp/i18n/de/editor.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
             "unsubmittedChanges": "Nicht abgesendete Änderungen vorhanden.",
             "submittingChanges": "Änderungen werden abgesendet.",
             "preparingForSubmit": "Absendung wird vorbereitet.",
-            "submit": "Absenden",
+            "submit": "Kompilieren und Ausführen",
```
Comment (krusche) [inline src/main/webapp/i18n/de/editor.json]: please revert this change!

File: src/main/webapp/i18n/en/editor.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
             "unsubmittedChanges": "There are unsubmitted changes.",
             "submittingChanges": "Submitting changes.",
             "preparingForSubmit": "Preparing for submit.",
-            "submit": "Submit",
+            "submit": "Compile and run",
```
Comment (krusche) [inline src/main/webapp/i18n/en/editor.json]: please revert this change and rather provide it as tooltip

File: src/main/webapp/i18n/de/modelingAssessmentEditor.json [hunk 3]
```diff
@@ -19,6 +19,7 @@
             "resultDismissed": "Deine Bewertung wurde nicht übernommen, da ein anderer Tutor bereits bewertet hat.",
             "updateAfterComplaintSuccessful": "Die Bewertung wurde erfolgreich aktualisiert.",
             "updateAfterComplaintFailed": "Das Aktualisieren der Bewertung ist fehlgeschlagen.",
+            "confirmSubmission": "Sind Sie sicher, dass Sie diese Bewertung einreichen möchten? Sie können es nachträglich nicht mehr ändern.",
```
Comment (krusche) [inline src/main/webapp/i18n/de/modelingAssessmentEditor.json]: Please use "du" in German

File: src/main/webapp/i18n/de/editor.json (hunk 1 above)
Comment (maxr96) [reply src/main/webapp/i18n/de/editor.json]: Sorry for that. Somehow it slipped in to my branch. Reverted the changed. What do you mean by tooltip?

File: src/main/webapp/i18n/en/editor.json (hunk 2 above)
Comment (maxr96) [reply src/main/webapp/i18n/en/editor.json]: Reverted

File: src/main/webapp/i18n/de/modelingAssessmentEditor.json (hunk 3 above)
Comment (maxr96) [reply src/main/webapp/i18n/de/modelingAssessmentEditor.json]: Right, changed that

**Comment by krusche** [comment]
Adding @SiggZ here: What do you think?

One small change request: Can you include a sentence in the popup like "you have not assessed all model elements"
